
==================[Вс 22.05.22 10:08:26]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[Вс 22.05.22 10:08:26]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306)
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at void System.Threading.Tasks.Task.ThrowIfExceptional(bool includeTaskCanceledExceptions)
=========================================================

==================[Вс 22.05.22 10:08:26]==================
System.IO.FileNotFoundException: Could not load file or assembly 'D:\RAGEMP\server-files\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'D:\RAGEMP\server-files\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[Вс 22.05.22 10:08:26]==================
System.IO.FileNotFoundException: Could not load file or assembly 'D:\RAGEMP\server-files\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'D:\RAGEMP\server-files\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[Вс 22.05.22 10:08:26]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
=========================================================

==================[Вс 22.05.22 10:08:26]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[Вс 22.05.22 10:08:26]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[Вс 22.05.22 10:08:26]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateEx
==================[Вт 24.05.22 11:03:11]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[Вт 24.05.22 11:03:11]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306)
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at void System.Threading.Tasks.Task.ThrowIfExceptional(bool includeTaskCanceledExceptions)
=========================================================

==================[Вт 24.05.22 11:03:11]==================
System.IO.FileNotFoundException: Could not load file or assembly 'D:\RAGEMP\server-files\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'D:\RAGEMP\server-files\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[Вт 24.05.22 11:03:11]==================
System.IO.FileNotFoundException: Could not load file or assembly 'D:\RAGEMP\server-files\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'D:\RAGEMP\server-files\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[Вт 24.05.22 11:03:11]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
=========================================================

==================[Вт 24.05.22 11:03:11]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[Вт 24.05.22 11:03:11]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[Вт 24.05.22 11:03:11]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateEx
==================[Пт 01.07.22 16:14:56]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[Пт 01.07.22 16:14:56]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306)
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at void System.Threading.Tasks.Task.ThrowIfExceptional(bool includeTaskCanceledExceptions)
=========================================================

==================[Пт 01.07.22 16:14:56]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[Пт 01.07.22 16:14:56]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[Пт 01.07.22 16:14:56]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
=========================================================

==================[Пт 01.07.22 16:14:56]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[Пт 01.07.22 16:14:56]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[Пт 01.07.22 16:14:56]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exceptio
==================[Пт 01.07.22 16:15:29]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[Пт 01.07.22 16:15:29]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306)
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at void System.Threading.Tasks.Task.ThrowIfExceptional(bool includeTaskCanceledExceptions)
=========================================================

==================[Пт 01.07.22 16:15:29]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[Пт 01.07.22 16:15:29]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[Пт 01.07.22 16:15:29]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
=========================================================

==================[Пт 01.07.22 16:15:29]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[Пт 01.07.22 16:15:29]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[Пт 01.07.22 16:15:29]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exceptio
==================[Пт 01.07.22 16:16:10]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[Пт 01.07.22 16:16:10]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306)
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at void System.Threading.Tasks.Task.ThrowIfExceptional(bool includeTaskCanceledExceptions)
=========================================================

==================[Пт 01.07.22 16:16:10]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[Пт 01.07.22 16:16:10]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\vlad\Desktop\SunriseDM\dotnet\resources\freemode\Debug\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[Пт 01.07.22 16:16:10]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
=========================================================

==================[Пт 01.07.22 16:16:10]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[Пт 01.07.22 16:16:10]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[Пт 01.07.22 16:16:10]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exceptio